mittab/libs/tab_logic.py

```diff
@@ -129,6 +129,25 @@ def pair_round():
                 # If there are no teams all down, give the bye to a one down team.
                 if bracket == 0:
                     byeint = len(list_of_teams[bracket])-1
+                    #Don't give the bye to a team who only has lenient NoShows
+                    if (TabSettings.get('lenient_late') + 1 >= current_round):
```

Comment:
I don't like this special exception from a code point of view or a tabulation point of view.

Code: We actually have code to "select" a bye twice, once in this part of the if block and again in the next section for dealing with 1-up teams getting the bye. If we keep this exception let's factor it out into a select_bye method or some such?

Tabulation: Byes are not a punishment (they're equivalent to a win by forfeit). If there are teams that have only had noshow loses, they should be just as eligible for a bye as anyone else.